File: sprint4/By languages/Java/lesson1code3.java
List<String> fruits = Arrays.asList("яблоко", "яблоко", "груша", "яблоко", "слива");
Map<String, Integer> fruitCount = new HashMap<>();
for (String fruit : fruits) {
    if (!fruitCount.containsKey(fruit)) {
        fruitCount.put(fruit, 0);
    }
    fruitCount.put(fruit, fruitCount.get(fruit) + 1);
}

File: sprint4/By languages/C++/lesson1code3.cpp
#include <iostream>
#include <map>
#include <vector>

using namespace std;

int main() {
    vector<string> fruits = {"яблоко", "яблоко", "груша", "яблоко", "слива"};
    map<string, int> fruitCount;
    for (string fruit : fruits) {
        if (fruitCount.find(fruit) == fruitCount.end()) {
            fruitCount[fruit] = 0;
        }
        fruitCount[fruit]++;
    }

    for (auto fruit : fruitCount) {
        cout << fruit.first << ": " << fruit.second << endl;
    }

    return 0;
}
